src/main/java/gov/cms/mat/cql_elm_translation/service/DataCriteriaService.java

```diff
@@ -135,10 +139,34 @@ private Set<String> getUsedDefinitionsFromMeasure(Measure measure) {
                           usedDefinitions.add(population.getDefinition());
                         }
                       });
+              if (group.getMeasureObservations() != null
+                  && !group.getMeasureObservations().isEmpty()) {
+                group
+                    .getMeasureObservations()
+                    .forEach(
+                        measureObservation -> {
+                          if (!measureObservation.getDefinition().isEmpty()) {
+                            usedDefinitions.add(measureObservation.getDefinition());
+                          }
+                        });
+              }
+              if (group.getStratifications() != null && !group.getStratifications().isEmpty()) {
```

Comment:
same thing regarding CollectionUtils.isEmpty

Reply:
switched it over